addons/mail/static/src/core/web/mail_composer_chatgpt.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="mail.MailComposerChatGPT">
+        <button class="btn btn-light"
+            t-if="props.record.data.can_edit_body"
+            t-on-click="onOpenChatGPTPromptDialogBtnClick">AI</button>
```

Comment:
I think translation in template requires text node to be at least 3 characters long.
To check but this is probably not taken into account by exported POT files.
Workaround is to do `_t()` in JS.

Reply:
I will use the `_t` function for that string :smiley: I will add a small comment indicating that it is a workaround for short strings.